modules/network-observability-virtualization-configuration.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * observability/network_observability/network-observability-secondary-networks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-virtualization-config_{context}"]
+= Configuring virtual machine (VM) secondary network interfaces for Network Observability
+You can observe network traffic on an OpenShift Virtualization setup by identifying eBPF-enriched network flows coming from VMs that are connected to secondary networks, such as through Open Virtual Network (OVN)-Kubernetes. Network flows coming from VMs that are connected to the default internal pod network are automatically captured by Network Observability.
+
+.Procedure
+. Get information about the virtual machine launcher pod by running the following command. This information is used in Step 5: 
++
+[source,terminal]
+----
+$ oc get pod virt-launcher-<vm_name>-<suffix> -n <namespace> -o yaml
+----
++
+[source,yaml]
+----
+$ oc get pod virt-launcher-fedora-aqua-fowl-13-zr2x9 -n my-vms -o yaml
```

Comment:
I'm assuming this line is supposed to be deleted since you show it in the code block above

Reply:
yep